client/src/pages/Profile/ProfileDetail/BookingCard/BookingCard.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import Grid from '@mui/material/Grid';
+import Paper from '@material-ui/core/Paper';
+import Box from '@mui/material/Box';
+import { DateRange } from '@mui/lab/DateRangePicker';
+import { FormikHelpers } from 'formik';
+import BookingForm from './BookingForm/BookingForm';
+import { Sitter } from '../sampleData';
+import useStyles from './useStyles';
+
+interface Props {
+  sitter: Sitter;
+}
+
+const BookingCard = ({ sitter }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [dateRange, setDateRange] = useState<DateRange<Date | null>>([null, null]);
+
+  const handleSubmit = (
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    { dateRange }: { dateRange: DateRange<Date> },
+    { setSubmitting, resetForm }: FormikHelpers<{ dateRange: DateRange<Date> }>,
+  ) => {
+    setSubmitting(false);
+    setDateRange([null, null]);
+    resetForm({ values: { dateRange: [null, null] } });
```

Comment:
This isn't entirely necessary but it's a good habit to leave a comment here, namely to hint/help out whoever does integration.

Reply:
Applied. Added comments on code